src/main/java/racingCar/utils/RandomUtil.java

```diff
@@ -0,0 +1,15 @@
+package racingCar.utils;
+
+import java.util.Random;
+
+public class RandomUtil {
+    private static final int MAX = 9;
+    private static Random random = new Random();
```

Comment:
```suggestion
    private static final Random random = new Random();
```

static 변수는 final을 붙여주는 것이 관례인데요
이러한 관례가 왜 생겼는지 고민해보아요 :)